res/controllers/midi-components-0.0.js

```diff
@@ -197,6 +198,8 @@
                 } else {
                     if (this.isLongPressed) {
                         this.inToggle();
+                    } else if (this.triggerOnRelease === true) {
```

Comment:
@Swiftb0y do you think truthy values assertion would be sufficient here? Happy to keep it if we think this is necessary, it's just that in general, checking if a boolean is true in a condition makes me itchy :rofl:

Reply:
nah, just this is fine:
```suggestion
                    } else if (this.triggerOnRelease) {
```

Reply:
Done.